autode/neb/original.py

```diff
@@ -244,32 +259,47 @@ def get_force(
         ) * hat_tau
 
         # ∇V(x)_i|_|_ = ∇V(x)_i - (∇V(x)_i•τ) τ
-        grad_perp = self.grad - np.dot(self.grad, hat_tau) * hat_tau
+        grad_perp = self.gradient - np.dot(self.gradient, hat_tau) * hat_tau
 
         # F_i = F_i^s|| -  ∇V(x)_i|_|_
         return f_parallel - grad_perp
 
+    @property
+    def gradient(self) -> Optional[np.ndarray]:
+        return None if self._grad is None else self._grad.flatten()
+
+    @gradient.setter
+    def gradient(self, value: Optional[np.ndarray]):
+        self._grad = None if value is None else value.flatten()
+
+    def _clear_energies_gradient_hessian(self) -> None:
+        """Skip any clearing energies"""
```

Comment:
Is this function supposed to do anything?

Reply:
nope – is meant to skip. but at the same time I don't remember why it's there.. if. I remove it then the tests still pass 😕. will delete and see if past me was wrong!